### PR TITLE
update README.md

### LINKED ISSUE #225: The readme is not consistent with the actual usage
The readme is not consistent with the actual usage. When charmhub got rid of the Docs tab, the content from discourse was moved to the Descriptions tab, making it super confusing as it now contains TWO overviews with entirely different content. 

We should
 
- Make sure the overview taken from the readme is up to date
- Incorporate everything valuable from the `Docs - Overview` discourse post and then delete that post.
- Incorporate everything valuable from the `Relations` discourse post and then delete that post.

### PR DESCRIPTION
**Pull request by Abuelodelanada**
## Issue

This PR fixes #225

> The readme is not consistent with the actual usage. When charmhub got rid of the Docs tab, the content from discourse was moved to the Descriptions tab, making it super confusing as it now contains TWO overviews with entirely different content. 
> 
> We should
>  
> - Make sure the overview taken from the readme is up to date
> - Incorporate everything valuable from the `Docs - Overview` discourse post and then delete that post.
> - Incorporate everything valuable from the `Relations` discourse post and then delete that post.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,391 @@
 
 ## Description
 
-[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. The Loki charmed operator deploys Loki in Kubernetes using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
+[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. This Loki charmed operator handles instanllation, scaling, configuration, optimisation, networking, service mesh, observability, and Day 2 operations specific to [Loki](https://grafana.com/docs/loki/latest/) using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
```
Comment (sed-i) [inline README.md:7]:
```suggestion
[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. This Loki charmed operator handles installation, scaling, configuration, optimisation, networking, service mesh, observability, and Day 2 operations specific to [Loki](https://grafana.com/docs/loki/latest/) using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
```

File: README.md [hunk 2]
```diff
@@ -4,32 +4,391 @@
...
+To set up a Promtail instance to work with Loki Charmed Operator please refer to [Configuring Promtail documentation](https://grafana.com/docs/loki/latest/clients/promtail/configuration/). Anyway the most important part is the `clients` section in Promtail config file, for instance:
+
+```yaml
+clients:
+  - url: http://<LOKI_ADDRESS>:3100/loki/api/v1/push
+```
+
+
+
+## Relations
```
Comment (sed-i) [inline README.md:107]: Now that we (sometimes) use [`description:` key in metadata.yaml](https://github.com/canonical/traefik-k8s-operator/blob/d8adda8e4eec41e270f403fbaa9ec56a74bd62d0/metadata.yaml#L54), should we incorporate this section in metadata instead?

**Review by sed-i — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Abuelodelanada) [reply README.md:107]: From my point of view we can add some description to metadata.yaml, but these explanations are to long to be there.

What do you think @simskij ?

Comment (simskij) [reply README.md:107]: I agree with you, José.

File: README.md [hunk 3]
```diff
@@ -4,32 +4,391 @@
...
+### Basic deployment
+
+Create a Juju model for your operator, say "observability"
 
 ```bash
 juju add-model observability
 ```
 
-Deploy the Loki Operator from Charmhub to the current model using the Juju command-line interface:
+The Loki Charmed Operator may be deployed using the Juju command line in a quite very way.
```
Comment (dimaqq) [inline README.md:23]: Is there perhaps a typo in `in a quite very way`?

Comment (dimaqq) [reply README.md:23]: Given the phrase `The way to relate Loki and Grafana is again very simple` lower in the document, perhaps the intention was to that Loki ... may be deployed ... in a simple way?

**Pull request merged**